Source/EVEClassLibrary/EVEClassLibrary/ConsoleMain.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Runtime;
+using System.IO;
+using Newtonsoft;
+
+namespace EVEClassLibrary
+{
+    /// <summary>
+    /// Main Console Class, Handles the Basic User Input
+    /// </summary>
+    public class ConsoleMain
+    {
+        /// <summary>
+        /// Initializing Console
+        /// </summary>
+        public static void init()
+        {
+            Creds();
+            Spacer(2);
+            StartInfo();
+            ParseCommand();
+        }
+
+        /// <summary>
+        /// Credits of the Library
+        /// </summary>
+        public static void Creds()
```

Comment:
Proposition: Move versionnumber ("v0.16") and author "DerToaster1138/Tanam Rotsuda") to project file and read these informations from the current assembly with System.Reflections Assembly class info